src/en/send.md

```diff
@@ -97,58 +97,100 @@ You can leave out this argument if your service only has one reply-to email addr
 Sending files by email is not enabled by default. To turn on this feature, [contact us](https://notification.canada.ca/contact).
 :::
 
-To send a file by email, you'll need to add a placeholder to the template then upload a file. The placeholder will contain a secure link to download the file. 
+### File types and size requirements
+You can upload PDF, CSV, .jpeg, .png, .odt, .txt, .rtf,  Microsoft Excel and Microsoft Word Document files. Your file must be smaller than 10MB.
```

Comment:
Maximum email size including **content** and **attachments** must be smaller than 10MB. The private link method shown below does not factor into calculated email size.